### ISSUE TITLE
Telescope is confused about (Javascript) repository root

### ISSUE THREAD

**Issue by githorse** [LunarVim/LunarVim#4511]
### Problem description

Just installed LunarVim. I have a Javascript/Typescript repository with a number of packages:

```
my-project/
  packages/
    packageA/
      src/
        FileIAmLookingAt.js
    packageB/
    packageC/
      src/
        FileIAmLookingFor.js
```

1. When I use the file finder (`<leader>f`), Telescope searches my entire repository (which is what I want), but the results falsely claim to be in the same package as the current file. For instance, If I'm looking at `fileIAmLookingAt.js` and I search for `LookingFor`, the Telescope match says: `../src/FileIAmLookingFor.js` instead of `../../packageC/src/FileIAmLookingFor.js`.

2. When I do a string search (`<leader>st`), Telescope only searches within the current package (i.e. the package of the file in the current buffer). I get no hits outside of the current package. (There's no issue with reporting the relative path correctly.)

**I expected**:
1. Both searches to take place at the repository root.
2. The file displayed paths to be relative to the current buffer (or, possibly, absolute from the root).

I didn't have either issue with my previous Telescope implementation (but I don't recall any details on setting that up). I'm sure I'm doing something wrong here -- any guidance is welcome!

### LunarVim version

release-1.3/neovim-0.9-d1c1bace

### Neovim version (>= 0.9.1)

0.9.5

### Terminal name

Terminal (MacOS)

### Operating system/version

macOS 14.2 (23C64)

### Steps to reproduce

See above.

**Comment by githorse** [comment]
I'm guessing that this is because my `cwd` is getting set to the (Javascript) package of the current file whenever I switch buffers. This doesn't seem to be [autochdir](https://vimdoc.sourceforge.net/htmldoc/options.html#'autochdir') doing this, best I can tell. I don't know what plugin is changing the directory or how to prevent it -- any ideas?

**Comment by githorse** [comment]
Got it: it's the [project.nvim plugin](https://github.com/ahmedkhalf/project.nvim). See https://github.com/LunarVim/LunarVim/discussions/2238. The immediate fix is to disable the plugin:

```
lvim.builtin.project.active = false
```

This turns off the code that was updating `cwd` whenever I changed buffers. Theoretically I can also set the project root to `".git"` and that would probably also work:

```
lvim.builtin.project.patterns = { ".git" }
```

(In my case I decided `project.nvim` wasn't adding much for my current workflow so just turned it off.)

But figuring this out was harder than it should have been, and the defaults here clearly didn't do what I wanted/expected in my Javascript monorepo, so I'll leave this issue open.
